apps/indexer/README.md

```
@@ -1,148 +1,148 @@
+        }
+        firstSeenLedger
+        lastSeenLedger
+      }
+    }
+  }
+}
+```
+
+Results:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)